cfpb/cfgov-refresh
# Alphabetize chart colors in Wagtail UI

## Changes

- Alphabetize color selection values in ChartBlock.

## Testing

1. Restart the local server.
2. Log into http://localhost:8000/admin/
3. Create a new Browse Page.
4. Add a Chart block to the page.
5. Set the required fields (if you try to save the page, it will notify you which are required).
6. Set the data source as `consumer-credit-trends/auto-loans/num_data_AUT.csv`.
7. See that color selection drop-down is alphabetize.
8. Preview the page and see that the chart renders.

## Screenshots

![screen shot 2018-08-29 at 10 04 34 am](https://user-images.githubusercontent.com/704760/44793129-8bcf2f80-ab73-11e8-82dc-5a643e5b9f57.png)
